### ISSUE TITLE
Data resolution for imputation 

### ISSUE THREAD

**Issue by weifangliu** [yezhengSTAT/scVI-3D#6]
Hello, I was wondering if scVI-3D can be used with 10kb scHi-C data for imputation?

**Comment by yezhengSTAT** [comment]
Yes, feel free to give it a try if the data is deeply sequenced and can be analyzed at this high resolution. 

Best,
Ye
